worlds/jakanddaxter/__init__.py

```diff
@@ -0,0 +1,265 @@
+
+        # Options drive which trade rules to use, so they need to be setup before we create_regions.
+        from .Rules import set_orb_trade_rule
+        set_orb_trade_rule(self)
+
+    # This will also set Locations, Location access rules, Region access rules, etc.
+    def create_regions(self) -> None:
+        from .Regions import create_regions
+        create_regions(self)
+
+        # from Utils import visualize_regions
+        # visualize_regions(self.multiworld.get_region("Menu", self.player), "jakanddaxter.puml")
+
+    # Helper function to reuse some nasty if/else trees.
+    def item_type_helper(self, item) -> Tuple[int, ItemClassification]:
+        # Make 101 Power Cells.
+        if item in range(jak1_id, jak1_id + Scouts.fly_offset):
+            classification = ItemClassification.progression_skip_balancing
+            count = 101
+
+        # Make 7 Scout Flies per level.
+        elif item in range(jak1_id + Scouts.fly_offset, jak1_id + Specials.special_offset):
+            classification = ItemClassification.progression_skip_balancing
+            count = 7
+
+        # Make only 1 of each Special Item.
+        elif item in range(jak1_id + Specials.special_offset, jak1_id + Caches.orb_cache_offset):
+            classification = ItemClassification.progression
+            count = 1
+
+        # Make only 1 of each Move Item.
+        elif item in range(jak1_id + Caches.orb_cache_offset, jak1_id + Orbs.orb_offset):
+            classification = ItemClassification.progression
+            count = 1
+
+        # Make N Precursor Orb bundles, where N is 2000 / bundle size.
+        elif item in range(jak1_id + Orbs.orb_offset, jak1_max):
+            classification = ItemClassification.progression_skip_balancing
+            count = 2000 // self.orb_bundle_size if self.orb_bundle_size > 0 else 0  # Don't divide by zero!
+
+        # Under normal circumstances, we will create 0 filler items.
+        # We will manually create filler items as needed.
+        elif item == jak1_max:
+            classification = ItemClassification.filler
+            count = 0
+
+        # If we try to make items with ID's higher than we've defined, something has gone wrong.
+        else:
+            raise KeyError(f"Tried to fill item pool with unknown ID {item}.")
+
+        return count, classification
+
+    def create_items(self) -> None:
+        for item_name in self.item_name_to_id:
+            item_id = self.item_name_to_id[item_name]
+
+            # Handle Move Randomizer option.
+            # If it is OFF, put all moves in your starting inventory instead of the item pool,
+            # then fill the item pool with a corresponding amount of filler items.
+            if item_name in self.item_name_groups["Moves"] and not self.options.enable_move_randomizer:
+                self.multiworld.push_precollected(self.create_item(item_name))
+                self.multiworld.itempool += [self.create_filler()]
```

Comment:
```suggestion
                self.multiworld.itempool.append(self.create_filler())
```
Probably quicker to not create a list for a single item